transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -188,7 +188,11 @@ private static void loadNativeLibrary() {
         if (!name.startsWith("linux")) {
             throw new IllegalStateException("Only supported on Linux");
         }
-        NativeLibraryLoader.load("netty-transport-native-epoll", PlatformDependent.getClassLoader(Native.class));
+        String []libraryNames = new String[]{
+          "netty-transport-native-epoll",
+          "netty_transport_native_epoll"
```

Comment:
I would need to experiment if the `maven-hawtjni-plugin` makes this configurable ([1] maybe useful), and if not we could always just mv the artifact after it is generated (via a `maven-antrun-plugin` execution).

[1] https://github.com/fusesource/hawtjni/blob/477c8ccac78c3695ebcf6299d8b201adb3394d34/maven-hawtjni-plugin/src/main/java/org/fusesource/hawtjni/maven/PackageJarMojo.java#L60